### PR TITLE
chore: Merge `next/main` into `main`

### PR DESCRIPTION
**Pull request by jimblanc**
#### Description of changes

Merging `next/main` into `main`. v5 has been backed up in the `v5-stable` support branch. **Merge with a merge commit.**

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by elorzafe — APPROVED** [review]
Lets go!

File: .github/workflows/callable-release-verification.yml [hunk 1]
```diff
@@ -7,16 +7,16 @@ jobs:
     uses: ./.github/workflows/callable-prebuild-amplify-js.yml
     with:
       runs_on: ubuntu-latest
-  prebuild-macos:
-    uses: ./.github/workflows/callable-prebuild-amplify-js.yml
-    with:
-      runs_on: macos-latest
+  # prebuild-macos:
```
Comment (stocaaro) [inline .github/workflows/callable-release-verification.yml:10]: Re-enabling this should speed things up a bit.

**Review by stocaaro — DISMISSED** [review]
LGTM. We've reviewed all of these parts and pieces independently and tested this. g2g 👍

File: .github/workflows/callable-release-verification.yml (hunk 1 above)
Comment (jimblanc) [reply .github/workflows/callable-release-verification.yml:10]: Ack, we can change on `main` once merged

**Review by stocaaro — APPROVED** [review]

**Review by david-mcafee — APPROVED** [review]

**Pull request closed without merging**